apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1.sln

```diff
@@ -60,4 +60,7 @@ Global
 	GlobalSection(SolutionProperties) = preSolution
 		HideSolutionNode = FALSE
 	EndGlobalSection
+	GlobalSection(ExtensibilityGlobals) = postSolution
+		EnterpriseLibraryConfigurationToolBinariesPathV6 = packages\EnterpriseLibrary.TransientFaultHandling.6.0.1304.0\lib\portable-net45+win+wp8
```

Comment:
Just to be clear, one of the test libraries will be including topaz (transient fault app block).  Not sure it was done correctly here, but it will be showing up as a nuget package for the test prj.